README.md

```diff
@@ -11,6 +11,7 @@ To start up a local environment of the Northlight UI framework docs:
 3) Run `yarn`.
 4) Open one teminal, navigate to the /docs directory, and run `yarn nx dev`.
 5) Open another terminal, navigate to the /framework directory, and `yarn watch`.
+6) Open a third terminal, navigate to the /docs directory and run `yarn transpile:watch`
```

Comment:
??

Reply:
We have to do this, unless we cant to re-transpile manually to update the comments that forms the examples